client/src/translate/translateController.js

```diff
@@ -2,8 +2,6 @@ import angular from 'angular';
 
 export default angular.module('translate')
     .controller('TranslateController', ['$translate', function ($translate) {
-        this.selectedLanguage = $translate.use();
-
         this.changeLanguage = (key) => {
             $translate.use(key);
             this.selectedLanguage = key;
```

Comment:
I guess you should remove this line as well?

Reply:
Updated.